Source/SwiftLintBuiltInRules/Rules/Lint/UnhandledThrowingTaskRule.swift

```diff
@@ -0,0 +1,260 @@
+import SwiftSyntax
+
+struct UnhandledThrowingTaskRule: ConfigurationProviderRule, SwiftSyntaxRule {
+    var configuration = SeverityConfiguration(.error)
+
+    static let description = RuleDescription(
+        identifier: "unhandled_throwing_task",
+        name: "Unhandled Throwing Task",
+        description: "Errors thrown inside this task are not handled, which may be unexpected. " +
```

Comment:
Could you please use a multiline string literal here? This also fixes some spacing issues in the string.

```swift
description: """
    Errors thrown inside this task are not handled, which may be unexpected. \
    Handle errors inside the task, or use `try await` to access the Tasks value and handle errors. \
    See this forum thread for more details: \
    https://forums.swift.org/t/task-initializer-with-throwing-closure-swallows-error/56066
    """,
```

Reply:
Ah yeah, good catch - was just following what I saw in another rule :)